test/images/volumes-tester/rbd/Dockerfile

```diff
@@ -15,16 +15,17 @@
 # CEPH all in one
 # Based on image by Ricardo Rocha, ricardo@catalyst.net.nz
 
-FROM fedora
+FROM fedora:26
 
 # Base Packages
-RUN yum install -y wget ceph ceph-fuse strace && yum clean all
+RUN yum install -y wget strace psmisc procps-ng ceph ceph-fuse && yum clean all
```

Comment:
this is failing, and I'm not sure how to resolve:
```
...
error: policycoreutils-newrole-2.6-6.fc26.x86_64: install failed
...
Error unpacking rpm package policycoreutils-newrole-2.6-6.fc26.x86_64                                      
policycoreutils-newrole-2.6-6.fc26.x86_64 was supposed to be installed but is not! 
```

Reply:
This is odd, simple `make image` in test/images/volumes-tester/rbd works for me. Can it be a stalled repo? Post whole `make` output.